frb_example/dart_minimal/lib/main.dart

```diff
@@ -1,8 +1,11 @@
 import 'package:frb_example_dart_minimal/src/rust/api/minimal.dart';
 import 'package:frb_example_dart_minimal/src/rust/frb_generated.dart';
 
+final LOGGER = FRBLogger.init_logger();
+
 // If you are developing a binary program, you may want to put it in `bin/something.dart`
 Future<void> main() async {
   await RustLib.init();
-  print('Call Rust and get: 100+200 = ${await minimalAdder(a: 100, b: 200)}');
+  LOGGER.info(
```

Comment:
maybe we can make the LOGGER *only* used from Rust. If some Dart users want to do logging, just follow the https://pub.dev/packages/logging doc and create another new logger.

Reply:
I refactored it now so that a user can just do that. 
Though instead of `final LOGGER = Logger.('my logger');` he would call `final LOGGER = FRBLogger.init_logger();` at least once.
As a bonus he doesn't need to add the logging package as a dependency.

Reply:
(I will check this in the next round review)

Reply:
Again it would be great to avoid such extra initialization (discussed below)